com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GraphicsUtil.java

```diff
@@ -14,80 +14,62 @@
 
 package com.dynamo.bob.pipeline;
 
-import java.util.List;
-import java.nio.ByteBuffer;
-import java.nio.ByteOrder;
-
```

Comment:
Changes in Bob:
* Changed how we store project data in `graphics_ddf.proto`.
  * The `int_values` and `uint_values` were merged into a single `long_values` field that covers the entire signed and unsigned 32-bit integer range.
  * The `float_values` field was changed to `double_values`. The editor does all math in doubles, and this enables us to add support for double attributes in the future.
  * Normalized values are stored as normalized doubles in `double_values` regardless of binary type. This allows users to change the binary type in the material without breaking overrides in sprites.
* We no longer support pre-provided byte values for `VertexAttributes` in Bob, because the editor doesn't, and these would be stripped when saving anyway.